tests/src/test_cases/test_bugfix.c

```diff
@@ -0,0 +1,29 @@
+#if LV_BUILD_TEST
+#include "../lvgl.h"
+
+#include "unity/unity.h"
+
+void test_bugfix_for_2522(void);
+
+void test_bugfix_for_2522(void)
```

Comment:
We need to decide on the grouping for these. IMO, the test should be named `test_arc_angles_when_reversed` and should be in `test_arc.c`. (There can be a comment before/inside the function mentioning #2522.)

Otherwise, we start having tests for a single widget type being spread across many files.

Reply:
I agree.

Reply:
Also agree, this file wasn't meant to be in this PR, I had it as a test for testing, and I noticed it was included in the commit after pushing it.

Reply:
Fixed in d425fe062